app/utils/pusher.js

```diff
@@ -28,6 +28,15 @@ TravisPusher.prototype.init = function (config, ajaxService) {
   if (config.host) {
     Pusher.host = config.host;
   }
+
+  if (ENV.pusher.debug) {
+    Pusher.log = function (message) {
+      if (window.console && window.console.log) {
```

Comment:
AFAIK `console.log` is sometimes not present when the console is not open? I might be mistaken though. I copy-pated it from the [pusher docs](https://pusher.com/docs/debugging#pusher_logging).